Gemfile.lock

```diff
@@ -58,10 +58,10 @@ GEM
     digest-crc (0.4.1)
     domain_name (0.5.20180417)
       unf (>= 0.0.5, < 1.0.0)
-    dotenv (2.7.1)
+    dotenv (2.7.2)
```

Comment:
Are we bumping these dependencies for a reason? Or is this just accidental?

Reply:
Completely accidental, I'm not too familiar with this build environment.